### ISSUE TITLE
'Current Season' option in list shows

### ISSUE THREAD

**Issue by wlcx** [UniversityRadioYork/MyRadio#311]

**Comment by Flump5281** [comment]
I'm assuming "current" is a season with scheduled timeslots, where at least one of those is before now and one after?

**Comment by wlcx** [comment]
Sounds about right, is there no concept of 'terms' in myradio?

**Comment by Flump5281** [comment]
There is, for but it's not currently used post-application and allocation.
I've just checked and there is a termid field for Seasons, however how do you then handle multiple seasons in a term? (Presenters are incredibly good at apply for three, four, five seasons a term, all identical)

**Comment by wlcx** [comment]
Just stick them all in the list?
What I was trying to achieve is a slightly more usable/fast list shows page - atm it's either 'my shows' or 'all shows ever', obvs, but most of the time the only shows of interest are the current year/term's. Any other ideas how that might be achievable?

**Comment by Flump5281** [comment]
"the list"? I thought you originally meant a button next to a show that opened the current season. The best person to ask these questions to is @Eiron - he's the one that actually uses it.

**Comment by wlcx** [comment]
Yeah, to clarify I'm meaning the scheduler. https://ury.org.uk/myury/Scheduler/allShows/ etc

**Comment by Flump5281** [comment]
Yeah - I got that. The bit of Scheduler I thought you meant was that page (and myShows) having another column that linked to something.

**Comment by wlcx** [comment]
Nope, just filtering the list

**Comment by Flump5281** [comment]
Cool - do it then. :P

**Comment by wlcx** [comment]
Ok, would the termid method work as above?

**Comment by Flump5281** [comment]
Should do.
